modules/harvest/src/Storage/HarvestHashesEntityDatabaseTable.php

```diff
@@ -267,4 +267,11 @@ protected function loadEntity(string $data_uuid): ?HarvestHashInterface {
     return NULL;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getTableName() {
+    return 'harvest_hashes';
```

Comment:
Follow the example of `get/setSchema()`:
```suggestion
    throw new \RuntimeException(__METHOD__ . ' not yet implemented.');
```